app/models/study.rb

```diff
@@ -1741,7 +1743,7 @@ def initialize_cluster_group_and_data_arrays(ordinations_file, user, opts={local
       raw_header_data = cluster_data.readline.encode('UTF-8', invalid: :replace, undef: :replace, replace: '').split(/[\t,]/).map(&:strip)
       raw_type_data = cluster_data.readline.encode('UTF-8', invalid: :replace, undef: :replace, replace: '').split(/[\t,]/).map(&:strip)
       header_data = self.sanitize_input_array(raw_header_data)
-      type_data = self.sanitize_input_array(raw_type_data)
+      type_data = self.sanitize_input_array(raw_type_data).map(&:downcase)
```

Comment:
What is this syntax? &:downcase

Reply:
This is Ruby shorthand for passing a block.  If we have an array of values called `array`:

`array.map(&:downcase) == array.map {|element| element.downcase}`

The `&` just means "for every element in this iterable object, call the following method", which is passed as a symbol.